hebiros/src/hebiros_subscribers_physical.cpp

```diff
@@ -346,14 +365,20 @@ void HebirosSubscribersPhysical::addEffortGainsCommand(GroupCommand* group_comma
 
 void HebirosSubscribersPhysical::feedback(std::string group_name, const GroupFeedback& group_fbk) {
 
-  std::shared_ptr<HebirosGroupPhysical> group = HebirosGroupPhysical::getGroup(group_name);
+  // TODO: replace with better abstraction later
+  HebirosGroupPhysical* group = dynamic_cast<HebirosGroupPhysical*>
```

Comment:
Remember that we will want to work towards getting rid of these dynamic_casts :)